tests/unit/test_support_centre_handlers.py

```diff
@@ -0,0 +1,423 @@
+from unittest import mock
+from aiohttp.test_utils import unittest_run_loop
+from aiohttp.client_exceptions import ClientConnectionError
+from aioresponses import aioresponses
+
+from . import RHTestCase
+
+import urllib.parse
+
+
+class TestSupportCentre(RHTestCase):
```

Comment:
possibly add some extra tests for bank holidays and census saturday/sunday

Reply:
Added extra checks in single and multiple tests to check for open and close of all days (across both sets of tests)